.gitignore

```diff
@@ -68,3 +68,7 @@ pids
 
 # Diagnostic reports (https://nodejs.org/api/report.html)
 report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
+
+# just junior dev stuff
+ztestfile
```

Comment:
If you have a local file, in your own environment, that doesn't necessarily impact all other devs, you can always just add that file you your own local `/.git/info/exclude` file for that given repo on your own machine, then it doesn't pollute the `.gitignore` file for everyone else.

https://luisdalmolin.dev/blog/ignoring-files-in-git-without-gitignore/#:~:text=To%20ignore%20untracked%20files%2C%20you,tracking%20any%20(untracked)%20file.